ante/ante.go

```diff
@@ -1,12 +1,13 @@
 package ante
 
 import (
+	errorsmod "cosmossdk.io/errors"
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	"github.com/cosmos/cosmos-sdk/x/auth/ante"
 	govkeeper "github.com/cosmos/cosmos-sdk/x/gov/keeper"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	gaiaerrors "github.com/cosmos/gaia/v10/types/errors"
```

Comment:
```suggestion
	gaiaerrors "github.com/cosmos/gaia/v11/types/errors"
```

Please change all occurances. V10 is a previous version, so the tooling will try to pull `gaia/v10` and break the build